datafusion/common/src/scalar.rs

```diff
@@ -3458,6 +3436,7 @@ impl ScalarType<i64> for TimestampNanosecondType {
 }
 
 #[cfg(test)]
+#[cfg(feature = "parquet")]
```

Comment:
Why do we have this?

Reply:
To enable running test. See #8250

Reply:
I am also puzzled by this but haven't spent the time to figure out what in `scalar.rs` is referring to parquet

Reply:
I think we should remove this cfg prior to merging

```suggestion
```